pom.xml

```diff
@@ -42,12 +42,28 @@
       <version>4.5.1</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>3.4.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <version>4.5.1</version>
+    </dependency>
+
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
       <version>2.15.2</version>
     </dependency>
 
+    <dependency><groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java3</artifactId>
+      <version>4.4.0</version></dependency>
```

Comment:
Use the last version of the library. You can check it in the maven website: 

https://mvnrepository.com/artifact/io.vertx/vertx-rx-java3/4.5.1

Check that you are using the last version on each dependency